tools/packaging/kata-deploy-cc/scripts/kata-deploy.sh

```diff
@@ -272,7 +272,7 @@ function remove_artifacts() {
 		/opt/confidential-containers/bin/kata-runtime \
 		/opt/confidential-containers/bin/kata-collect-data.sh \
 		/opt/confidential-containers/bin/qemu-system-x86_64 \
-		/opt/confidential-containers/bin/qemu-system-x86_64-snp \
+		/opt/confidential-containers/bin/qemu-system-x86_64-snp-experimental \
```

Comment:
Could we just take the same approach as `main` does:
```
function remove_artifacts() {
    echo "deleting kata artifacts"
    rm -rf /opt/kata/
}
```
and delete the whole directory, or does the pre-install & post-uninstall using it cause issues with that?

Reply:
The problem on deleting the whole directory is that we'll try to delete the `containerd` binary that's also located there, and any other binary that folks would end up putting there as part of their modified pre-install / post-uninstall containers. :-/

Reply:
But one thing that can be done, but not as part of this PR, is a test to ensure that everything that's part of the image is being into account to be cleaned up ... that would not be hard to implement and that would be a very nice way to provent it happening in the future.

Reply:
Partially.

When we get rid of the containerd fork, **I imagine** that we'll still be relying on a speciifc version of containerd, which will have the patches proposed by the community in order to better support CoCo.  If that's really the case, we'll  have to keep shipping that minimal version of containerd for a reasonable amount of time (preferrably till it gets EOD and we don't have to care about folks using a non-supported version of containerd anymore).